scipy/stats/tests/test_continuous_basic.py

```diff
@@ -828,3 +828,13 @@ def test_kappa3_array_gh13582():
                     for moment in moments])
     res2 = np.array(stats.kappa3.stats(shapes, moments=moments))
     npt.assert_allclose(res, res2)
+
+
+def test_kappa4_array_gh13582():
+    h = [0.5, 1.5, 2.5, 3.5, 4.5]
+    k = [0.5, 1, 1.5, 2, 3]
```

Comment:
You'd want the values here to test all the different logical conditions, so some of these should be negative. Consider also using broadcasting to get a good mix of things:
```suggestion
    h = np.array([-1, -0.5, 0, 1])
    k = np.array([-1, -0.5, 0, 1])[:, None]
```

Reply:
This was marked resolved, but it wasn't implemented. It turns out that it (broadcasting) doesn't work as it should. For reference, 
```python3
import numpy as np
from scipy import stats
h = np.array([-1, -0.5, 1])
k = np.array([-1, -0.5, 0, 1])[:, None]
moments = 'mvsk'
print(np.array(stats.kappa4.stats(h, k, moments=moments)).shape)
print(np.array(stats.trapezoid.stats(h, k, moments=moments)).shape)
```
prints
```
(4, 3)
(4, 4, 3)
```

Reply:
> turns out that it (broadcasting) doesn't work as it should.

Hmm, broadcasting isn't working. Need to look more into it

Reply:
Added a test for it.